apps/hexagon_proxy_rpc/rpc_env.cc

```diff
@@ -0,0 +1,321 @@
+/*
```

Comment:
How is this loaded?  Would it make sense to have this under src/runtime/hexagon/rpc?

Reply:
`rpc_env.cc` is a hexagon version of [rpc_env.cc](https://github.com/apache/tvm/blob/main/apps/cpp_rpc/rpc_env.cc) from apps/cpp_rpc.  In this case, all other sources from the C++ RPC server are reused.  As noted in the description above we are working on a solution to run a minrpc server directly on hexagon which will provide more general functionality (e.g. support for tuning). This minrpc implementation will land in `src/runtime/hexagon/rpc as you suggest`. Until then, we are submitting this prototype forwarding RPC server application for use in development.

Reply:
cc @mehrdadh who is working on the minrpc server implementation.

Reply:
I agree with @csullivan. Let's keep these files under `app` and he minRPC server under `src`